File: cpp/171.cpp
class Solution {
public:
    int titleToNumber(string& c) {
        int s = 0, n = c.length();
        long long m = 1;
        for (int i = n - 1; i >= 0; i--, m *= 26)
            s += m * (c[i] - 64);
        return s;
    }
};

File: javascript/171.js
/**
 * @param {string} c
 * @return {number}
 */
var titleToNumber = function (c) {
    let s = 0, n = c.length, m = 1;
    for (let i = n - 1; i >= 0; i--, m *= 26)
        s += m * (c.codePointAt(i) - 64);
    return s;
};
